CodeGen/tests/test_codegen_on_xeon.sh

```diff
@@ -20,6 +20,7 @@ function build_docker_images() {
 
     cd $WORKPATH/docker/ui
     docker build --no-cache -t opea/codegen-ui:latest -f docker/Dockerfile .
+    docker build --no-cache --build-arg BACKEND_SERVICE_ENDPOINT=http://${ip_address}:7778/v1/codegen -t opea/codegen--react-ui:latest -f docker/Dockerfile.react .
```

Comment:
fix the typo: `opea/codegen--react-ui:latest`

Reply:
fixed typo